pkg/capabilities/pb/capabilities_helpers.go

```diff
@@ -1,9 +1,101 @@
 package pb
 
+import (
+	"errors"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
 const (
 	CapabilityTypeUnknown   = CapabilityType_CAPABILITY_TYPE_UNKNOWN
 	CapabilityTypeTrigger   = CapabilityType_CAPABILITY_TYPE_TRIGGER
 	CapabilityTypeAction    = CapabilityType_CAPABILITY_TYPE_ACTION
 	CapabilityTypeConsensus = CapabilityType_CAPABILITY_TYPE_CONSENSUS
 	CapabilityTypeTarget    = CapabilityType_CAPABILITY_TYPE_TARGET
 )
+
+func MarshalCapabilityRequest(req capabilities.CapabilityRequest) ([]byte, error) {
+	return proto.Marshal(CapabilityRequestToProto(req))
```

Comment:
Nit: I would inline `CapabilityRequestToProto(req)` here or move `proto.Marshal` into `CapabilityRequestToProto` (pref for former). Seems like the function split is unnecessary and makes it harder to read. Same for the other three functions below, i.e., I'd expect 4 functions total - Marshal/Unmarshal * Request/Response

Reply:
ToProto/FromProto are used in capabilities.go after this refactor and they don't need un/marshaling to bytes. Marshal/Unmarshal will be useful in my next core PR.